runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -959,6 +959,11 @@ private void expand(@Nullable NavigationButton button)
 
 		int width = panel.getWrappedPanel().getPreferredSize().width;
 		int expandBy = pluginPanel != null ? pluginPanel.getWrappedPanel().getPreferredSize().width - width : width;
+
+		if (pluginPanel != null)
+		{
+			pluginPanel.onDeactivate();
+		}
```

Comment:
A comment helps clarify this bit of code.
```suggestion
		// Deactivate previously active panel
		if (pluginPanel != null)
		{
			pluginPanel.onDeactivate();
		}

```